impl/vfs/hdfs/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -21,25 +22,25 @@
   </bean>
 
   <bean class="org.pentaho.big.data.impl.vfs.hdfs.HDFSFileProvider" scope="singleton">
-      <argument ref="hadoopFileSystemService"/>
-      <argument ref="namedClusterService"/>
-      <argument ref="maprfsFileNameParser" />
-      <argument value="escalefs" type="java.lang.String"/>
+    <argument ref="hadoopFileSystemService"/>
+    <argument ref="namedClusterService"/>
+    <argument ref="maprfsFileNameParser" />
+    <argument value="escalefs" type="java.lang.String"/>
     <argument ref="metastoreLocatorOsgi" />
   </bean>
 
   <bean class="org.pentaho.big.data.impl.vfs.hdfs.HDFSFileProvider" scope="singleton">
     <argument ref="hadoopFileSystemService"/>
     <argument ref="namedClusterService"/>
-    <argument ref="maprfsFileNameParser" />
```

Comment:
why did you remove the maprfsFileNameParser here

Reply:
erm, it's not removed, it's been replaced with the Azure parser for WASB and ABFS scheme's which are Azure HDI Shim specific.